easybuild/easyblocks/s/suitesparse.py

```diff
@@ -70,24 +72,32 @@ def configure_step(self):
             'LAPACK': os.getenv('LIBLAPACK_MT'),
         }
 
+        # Get METIS or ParMETIS settings
         metis = get_software_root('METIS')
         parmetis = get_software_root('ParMETIS')
         if parmetis:
             metis_path = parmetis
-            metis_libs = ' '.join([
-                os.path.join(parmetis, 'lib', 'libparmetis.a'),
-                os.path.join(parmetis, 'lib', 'metis.a'),
-            ])
+            metis_include = os.path.join( parmetis, 'include')
+            metis_libs = os.path.join(parmetis, get_software_libdir('ParMETIS'), 'libmetis.a')
```

Comment:
I don't think `SuiteSparse` knows anything about `ParMETIS`. It cannot link to it. It only depends on `METIS`. And depending on `ParMETIS` is an alternative way to provide `METIS`.

A look at the README.txt of `SuiteSparse` only mentions `METIS`, never `ParMETIS`.